pom.xml

```diff
@@ -85,6 +91,26 @@
                 <artifactId>mockito-core</artifactId>
                 <version>5.3.1</version>
             </dependency>
+            <dependency>
+                <groupId>io.specto</groupId>
+                <artifactId>hoverfly-java</artifactId>
+                <version>0.15.0</version>
```

Comment:
could you please move all versions in the `<properties>` section to be consistent with other dependencies?